src/7tv-emotes/modules/avatars.js

```diff
@@ -0,0 +1,138 @@
+export default class Avatars extends FrankerFaceZ.utilities.module.Module {
```

Comment:
It might be worth giving the class names a prefix? I'm not sure this matters, but just in case since you're extending their module class